src/Symfony/Component/Form/CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 CHANGELOG
 =========
 
+3.4.0
+-----
+
+ * deprecated `ChoiceLoaderInterface` implementation in `TimezoneType`
```

Comment:
Well.. it's consistent with the current format :)

Shall add the new options here as well :+1: